skorch/callbacks/training.py

```diff
@@ -84,20 +87,36 @@ class Checkpoint(Callback):
 
       >>> cb = Checkpoint(f_params="params_{last_epoch[epoch]}.pt")
 
-    f_history : file-like object, str, None (default=None)
+    f_optimizer : file-like object, str, None (default='optimizer.pt')
+      File path to the file or file-like object where the optimizer
+      state should be saved. Pass ``None`` to disable saving
+      model parameters.
+
+      Supports the same format specifiers as ``f_params``.
+
+    f_history : file-like object, str, None (default='history.json')
       File path to the file or file-like object where the model
       training history should be saved. Pass ``None`` to disable
       saving history.
 
-      Supports the same format specifiers as ``f_params``.
-
     f_pickle : file-like object, str, None (default=None)
       File path to the file or file-like object where the entire
       model object should be pickled. Pass ``None`` to disable
       pickling.
 
       Supports the same format specifiers as ``f_params``.
 
+    fn_prefix: str (default='')
```

Comment:
Shouldn't this be called `f_prefix` for consistency? Otherwise sometimes we "abbreviate" filename to "f" and sometimes to "fn".

Reply:
I think `f` is a file-like object while `fn` is the name specifically. So it makes sense to have `f_*` for destinations/paths/file-likes and `fn_prefix` for a common prefix *if* `f_*` params are actually names.